cmd/booster-http/run.go

```diff
@@ -41,7 +41,7 @@ var runCmd = &cli.Command{
 		&cli.StringFlag{
 			Name:  "nitro-endpoint",
 			Usage: "the endpoint for the nitro server",
-			Value: "127.0.0.1:4007",
+			Value: "127.0.0.1:4007/api/v1",
```

Comment:
I think we have a bit of a mismatch between our ts and go rpc clients. The former seems to "know" about the /api/v1 suffix, whereas the latter needs to be told it.

What's the best way to resolve the difference?

Reply:
I noticed that as well. It seems like a full endpoint should be specified (instead of assuming the `api/v1` path. What do you think?

Reply:
Perhaps we should allow the user to pass in an api version (which fails if its an unsupported version), but we default to the latest version (v1 in this case) if the user does not provide a version when they initialize the client?

Reply:
That's a great long term approach. But perhaps overkill for now? Simpler options are to assume `api/v1` path or to require a full url (hostname/ip address, port and path).

Reply:
That makes sense to me. Here is a PR to update the typescript nitro-rpc-client to expect the full jsonrpc server url: https://github.com/statechannels/nitro-gui/pull/78